src/Bridge/Symfony/Bundle/DependencyInjection/Configuration.php

```diff
@@ -92,6 +92,27 @@ public function getConfigTreeBuilder()
                         ->end()
                     ->end()
                 ->end()
+
+                ->arrayNode('http_cache')
+                    ->addDefaultsIfNotSet()
+                    ->children()
+                        ->integerNode('max_age')->defaultNull()->info('Default value for the response max age.')->end()
+                        ->integerNode('shared_max_age')->defaultNull()->info('Default value for the response shared (proxy) max age.')->end()
+                        ->arrayNode('vary')
+                            ->defaultValue(['Content-Type'])
+                            ->prototype('scalar')->end()
+                            ->info('Default values of the "Vary" HTTP header.')
+                        ->end()
+                        ->booleanNode('public')->defaultNull()->info('To make all responses public by default.')->end()
+                        ->booleanNode('enable_tags')->defaultFalse()->info('Add cache tags to the response.')->end()
```

Comment:
it's not only about adding tags in response, it's also about cache invalidation.